backend/app/tests/utils/test_content_parser.py

```
@@ -0,0 +1,470 @@
+
+        # Both should handle the content reasonably
+        assert isinstance(pdf_result, str)
+        assert isinstance(text_result, str)
+        assert len(pdf_result) > 0
+        assert len(text_result) > 0
+
+    def test_error_resilience(self):
+        """Test error handling and resilience"""
+        # Test with None input (should not crash)
+        try:
+            _none_result = self.parser._extract_text_from_html(None)
```

Flake8 (7.2.0)

[error] local variable '_none_result' is assigned to but never used

(F841)